{
  "description": "An issue was discovered in the Linux kernel through 4.17.10. There is an out-of-bounds access in __remove_dirty_segment() in fs/f2fs/segment.c when mounting an f2fs image.",
  "releases": {
    "bullseye": {
      "fixed_version": "4.19.9-1",
      "repositories": {
        "bullseye": "5.6.14-2"
      },
      "status": "resolved",
      "urgency": "not yet assigned"
    },
    "buster": {
      "fixed_version": "4.19.9-1",
      "repositories": {
        "buster": "4.19.118-2",
        "buster-security": "4.19.118-2+deb10u1"
      },
      "status": "resolved",
      "urgency": "not yet assigned"
    },
    "jessie": {
      "nodsa": "Hard to backport and low priority outside of Android",
      "nodsa_reason": "ignored",
      "repositories": {
        "jessie": "3.16.56-1+deb8u1",
        "jessie-security": "3.16.84-1"
      },
      "status": "open",
      "urgency": "not yet assigned"
    },
    "sid": {
      "fixed_version": "4.19.9-1",
      "repositories": {
        "sid": "5.7.6-1"
      },
      "status": "resolved",
      "urgency": "not yet assigned"
    },
    "stretch": {
      "fixed_version": "4.9.144-1",
      "repositories": {
        "stretch": "4.9.210-1",
        "stretch-security": "4.9.210-1+deb9u1"
      },
      "status": "resolved",
      "urgency": "not yet assigned"
    }
  },
  "scope": "local"
}